### DISCUSSION TITLE
Update - 4.30 pm (CET) confirmed / 4.00 pm or 4.30 pm (CET)

### DISCUSSION THREAD

**Discussion by xzhh** [espressopp/espressopp#469] [Team Posts]
Dear all

Last week we discussed about whether to shift to 4.30 pm for the next couple of weeks (Andre has a time conflict due to lectures). But I remember Luis might be also invited for this week's discussion. If so, we can as usual start from 4.00 pm.

Any ideas? (Especially from MPCDF people and also about the time shift in general)

Zhenhao

**Comment by XzzX** [comment]
Dear all,

I am ok with 4pm and 4:30pm.

Best
Sebastian

**Comment by mrampp** [comment]
I'm open for 16:00 or 16:30 (no preference).

Markus

On Wed, Nov 09 2022, Zhen-Hao Xu wrote:
> Dear all
> 
> Last week we discussed about whether to shift to 4.30 pm for the next couple of weeks (Andre has a time conflict due to lectures). But I remember Luis might be also invited for this week's discussion. If so, we can as usual start from 4.00 pm.
> 
> Any ideas? (Especially from MPCDF people and also about the time shift in general)
> 
> Zhenhao
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/orgs/espressopp/teams/discussion/discussions/15
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
best regards / schoene Gruesse,
Markus Rampp

--  
---------------------------------------------------------
Dr. Markus Rampp
Max Planck Computing and Data Facility (MPCDF)
Giessenbachstr. 2, D-85748 Garching bei Muenchen, Germany
Phone: +49-89-3299-1324           FAX: +49-89-3299-1301
E-Mail: ***@***.*** WWW: www.mpcdf.mpg.de
---------------------------------------------------------

**Comment by xzhh** [comment]
Okay, lets make it 4.30 pm. If any of us needs to leave early, we can adjust the topic session accordingly.
